spot-contracts/contracts/strategies/BasicFeeStrategy.sol

```diff
@@ -47,6 +53,11 @@ contract BasicFeeStrategy is IFeeStrategy, OwnableUpgradeable {
     /// @notice Fixed percentage of the rollover amount to be used as fee.
     int256 public rolloverFeePerc;
 
+    /// @notice Enables debasement of perp supply when the fee reserve is empty.
+    /// @dev When the fees to be paid out are negative, ie) paid from the user to the reserve
+    ///      this flag stops paying out more than the reserve balance through perp supply inflation.
+    bool public debasement;
```

Comment:
`debasement` -> `allowDebase`?

(Same for the event param and other places if we change it)